<launch>

  <arg name="namespace" default="detection"/>
  <arg name="packagename" default="shape_estimation"/>
  <arg name="input" default="/detection/lidar_detector/objects"/>
  <arg name="output" default="/detection/shape_estimation/objects"/>

  <node pkg="lidar_shape_estimation" type="lidar_shape_estimation" name="lidar_shape_estimation" output="screen">
    <remap from="input"                to="$(arg input)" />
    <remap from="objects"              to="$(arg output)" />
    <param name="bbox_target_frame"    value="velodyne"/>
    <param name="filter_res"           value="0.1"/>
  </node>

  <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="world_velodyne" args="0 0 5 0 0 0 1  velodyne world" /> -->

  <node pkg="detected_objects_visualizer" type="visualize_detected_objects" name="shape_estimation_visualization"
        output="screen" ns="/$(arg namespace)/$(arg packagename)"/>

</launch>
